### PR TITLE
Completed array-manipulation-1.

### PR DESCRIPTION
**Pull request by alexdenbaugh**
array-manipulation-1

### CODE REVIEW

File: array-manipulation-1/src/compact.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+//       };
+//     };
+//   };
+//   return newArray;
+// };
+
+function compact(array) {
+  var newArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (array[i] > 0 || array[i] < 0 || array[i] === true || (array[i] !== '' && typeof (array[i]) === typeof ('')) || (typeof (array[i]) === typeof ([]) && array[i] !== null)) {
```
Comment (codingcodymiller) [inline array-manipulation-1/src/compact.js:17]: You can check if a value is truthy by just passing the value directly into the condition of an if statement, like so: `if(array[i])`

File: array-manipulation-1/src/initial.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/* exported initial */
+function initial(array) {
+  var newArray = [];
+  for (var i = 0; i < array.length - 1; i++) {
+    newArray[i] = array[i];
```
Comment (codingcodymiller) [inline array-manipulation-1/src/initial.js:5]: Use `push` instead of assigning to indexes in an array

File: array-manipulation-1/src/tail.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+/* exported tail */
+function tail(array) {
+  var newArray = [];
+  for (var i = 1; i < array.length; i++) {
+    newArray[i - 1] = array[i];
```
Comment (codingcodymiller) [inline array-manipulation-1/src/tail.js:5]: Again, just use `push`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far!! please see my notes

**Comment by alexdenbaugh** [comment]
array-manipulation-1-revised

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**